src/renderers/webgl-fallback/nodes/GLSLNodeBuilder.js

```diff
@@ -636,9 +637,24 @@ ${ flowData.code }
 
 	}
 
+	enableExtension( name, mode, shaderStage = this.shaderStage ) {
```

Comment:
In GLSL it is not called `mode` but `behavior`.

see https://www.khronos.org/opengl/wiki/Core_Language_(GLSL)#Extensions